src/adapters/property-base-adapter/index.ts

```diff
@@ -56,6 +56,28 @@ export async function searchBuildings(
   }
 }
 
+type GetPropertiesResponse = components['schemas']['Property'][]
+
+export async function getProperties(
+  companyCode: string,
+  tract?: string
+): Promise<AdapterResult<GetPropertiesResponse, 'not-found' | 'unknown'>> {
+  try {
+    const fetchResponse = await client().GET('/properties', {
+      params: { query: { companyCode, tract } },
+    })
+
+    if (fetchResponse.data?.content) {
+      return { ok: true, data: fetchResponse.data.content }
+    }
+
+    return { ok: false, err: 'unknown' }
```

Comment:
You could throw something here so it ends up in the catch and the logging that happens there 🤷